cmd/zt_list_test.go

```diff
@@ -0,0 +1,162 @@
+package cmd
+
+import (
+	"github.com/Azure/azure-sdk-for-go/sdk/storage/azblob/container"
+	"github.com/Azure/azure-storage-azcopy/v10/common"
+	"github.com/stretchr/testify/assert"
+	"strings"
+	"testing"
+)
+
+func TestListVersions(t *testing.T) {
```

Comment:
we should also test running tally

Reply:
Running tally is on in both tests at line 51 and 123. I can write additional tests covering running tally if needed!

Reply:
maybe we can add one with a mixed variety of blobs all together
some with versions, some without. some with 1 version, some with multiple versions, etc

Reply:
I can update TestListVersionsMultiVersions so that one blob has 2 versions. Right now, TestListVersionsMultiVersions tests: 2 blobs with 1 version and 2 without.

Reply:
Updated! MultiVersions now tests 1 blob with 2 versions, 1 blob with 1 version, 2 blobs without versions.